File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/LingkaranLatihan.java
package id.ac.uph.pbo.pertemuan01;

public class LingkaranLatihan {
    private double radius;
    public LingkaranLatihan(double radius) { this.radius = radius; }
    public double hitungLuas()     { return Math.PI * radius * radius; }
    public double hitungKeliling() { return 2 * Math.PI * radius; }
    public void cetak() {
        System.out.printf("  Lingkaran (r=%.1f): Luas=%.2f, Keliling=%.2f%n",
                radius, hitungLuas(), hitungKeliling());
    }
}

File: PYTHON/Pertemuan 01/LingkaranLatihan.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
import math

class LingkaranLatihan:
    def __init__(self, radius: float):
        self.radius = radius

    def hitung_luas(self) -> float:
        return math.pi * self.radius * self.radius

    def hitung_keliling(self) -> float:
        return 2 * math.pi * self.radius

    def cetak(self):
        print(f"  Lingkaran (r={self.radius:.1f}): Luas={self.hitung_luas():.2f}, Keliling={self.hitung_keliling():.2f}")

